### ISSUE TITLE
Abra2 creating large (around 100bp) deletions to alignment

### ISSUE THREAD

**Issue by rchapman2000** [mozack/abra2#54]
Hello,

I have been using abra2 for viral reference based assembly for a while now and have not had any issues. However, recently I have noticed that in a few of my assemblies, there extremely large gaps being added in to the alignment by abra2. It appears that the tool is taking a few bases from the ends of a read and moving to a distant position. It is also odd because these regions do not contain any mismatches from the reference...

Below are screenshots of my alignment before and after running abra2.

Before:
![image](https://user-images.githubusercontent.com/50974682/223209753-ec60aca9-2e85-4148-8370-446b5fdd69a0.png)

After:
![image](https://user-images.githubusercontent.com/50974682/223210251-241c3277-3a0a-4848-804d-fce5a9ea9a5a.png)

I am currently running abra2 version 2.23 installed via conda.

Let me know what other information I can provide! Thank you!

**Comment by wldgkl2014** [comment]
We are also encountering a similar problem. How did you solve it?

**Comment by rchapman2000** [comment]
> We are also encountering a similar problem. How did you solve it?

No, I don't think so. To be honest, I have not been actively working on genome assembly for several years. Looking back on my pipelines, I removed this tool from them.
